server/master/src/main/java/org/apache/accumulo/master/FateServiceHandler.java

```diff
@@ -162,10 +162,11 @@ public void executeFateOperation(TInfo tinfo, TCredentials c, long opid, FateOpe
               TableOperationExceptionType.OTHER,
               "Expected at least " + SPLIT_OFFSET + " arguments, saw :" + arguments.size());
         }
-        String tableName = validateTableNameArgument(arguments.get(0), tableOp, NOT_SYSTEM);
+        String tableName = validateNewTableNameArgument(arguments.get(0), tableOp, NOT_SYSTEM);
         TimeType timeType = TimeType.valueOf(ByteBufferUtil.toString(arguments.get(1)));
         InitialTableState initialTableState =
             InitialTableState.valueOf(ByteBufferUtil.toString(arguments.get(2)));
+        log.info("Init Table State: " + ByteBufferUtil.toString(arguments.get(2)));
```

Comment:
Was this added for debugging?

Reply:
@keith-turner Yes, it was. I need to take that out.